src/Maestro/tests/Maestro.ScenarioTests/TestHelpers.cs

```diff
@@ -40,15 +48,17 @@ void WriteOutput(string message)
             {
                 StartInfo = psi
             };
+
             var tcs = new TaskCompletionSource<bool>(TaskCreationOptions.RunContinuationsAsynchronously);
             process.EnableRaisingEvents = true;
             process.Exited += (s, e) => { tcs.TrySetResult(true); };
             process.Start();
 
             Task<bool> exitTask = tcs.Task;
+            Task stdin = Task.Run(async () => { process.StandardInput.Write(input); process.StandardInput.Close(); });
```

Comment:
I'm... confused about this call. There isn't anything async in here... Are we not able to just call "process.StandardInput.Write(input)" here without the need for the Task.Run?

Reply:
I pulled the async (it was extraneous in any case), but this is the set up that Alex recommended. I'll try it without a Task, but my vague understanding was that the task is needed because of the way the output is handled.